brandontjd/esd-mentino
# Add bubble email service

#### Summary
Add email microservice that listens to rabbitMQ and sends email depending on the event name
(Will only have `mentor_joined` event)

#### How to test
1. Place the new `.env` file in the `/backend`
2. `docker-compose up --build`
3. Use Postman Collection in `/backend/user` to sign up with your own email and log in to get a token
4. With the same Postman Collection, place the token in the bearer authorization for `/protected_endpoint`
5. You should receive the notification in your registered email

#### How to replicate to trigger in the backend
Refer to `protected_endpoint.py`, there is a sample snippet
